mods/ra/weapons/other.yaml

```diff
@@ -185,9 +185,12 @@ TTankZap:
 
 DogJaw:
 	ValidTargets: Infantry
-	ReloadDelay: 10
-	Range: 3c0
+	ReloadDelay: 100
+	FireDelay: 200
 	Report: dogg5p.aud
+	Range: 1c512
+	Projectile: Bullet
+		Speed: 1c682
 	Warhead@1Dam: SpreadDamage
```

Comment:
The problem is that only the `*Zap`s and `Missile` tracks the target, (and `Missile` will never hit it due to CloseEnough). Even with a direct-damage warhead present, it will be still useless because all projectiles will miss it - some doesn't even use `GuidedTarget` to begin with!